tldw_Server_API/app/api/v1/API_Deps/Slides_DB_Deps.py

```
@@ -0,0 +1,88 @@
+
+_MAX_CACHED_SLIDES_DB = 20
+_slides_db_lock = threading.Lock()
+_slides_db_instances: Dict[str, SlidesDatabase] = {}
+
+
+def _get_slides_db_path_for_user(user_id: int) -> Path:
+    return DatabasePaths.get_slides_db_path(user_id)
+
+
+async def get_slides_db_for_user(
+    current_user: User = Depends(get_request_user),
```

Ruff (0.14.13)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)